apps/test/unit/lib/kits/maker/LedTest.js

```diff
@@ -2,7 +2,7 @@
 import {expect} from '../../../../util/deprecatedChai';
 import sinon from 'sinon';
 import five from '@code-dot-org/johnny-five';
-import makeStubBoard from './makeStubBoard';
+import {makeStubBoard} from './makeStubBoard';
```

Comment:
What does putting this in curly braces do?

Reply:
'makeStubBoard' was exclusive to the CP and so it was the default export of this file - so it didn't need braces. However, we needed a stubBoard for MB, so I updated the 'makeStubBoard' file to not have a default export and then added an exportable class for the MB. Since the function 'makeStubBoard' is not longer the default export, it needs to go in braces.